app/features/top-search/SplatoonPlayerRepository.server.ts

```diff
@@ -7,3 +8,72 @@ export function unlinkPlayerByUserId(userId: number) {
 		.where("SplatoonPlayer.userId", "=", userId)
 		.execute();
 }
+
+function selector() {
+	return db
+		.selectFrom("XRankPlacement")
+		.select([
+			"XRankPlacement.id",
+			"XRankPlacement.weaponSplId",
+			"XRankPlacement.name",
+			"XRankPlacement.power",
+			"XRankPlacement.rank",
+			"XRankPlacement.month",
+			"XRankPlacement.year",
+			"XRankPlacement.region",
+			"XRankPlacement.playerId",
+			"XRankPlacement.mode",
+		])
+		.leftJoin("SplatoonPlayer", "XRankPlacement.playerId", "SplatoonPlayer.id")
+		.leftJoin("User", "SplatoonPlayer.userId", "User.id")
+		.select(["User.discordId", "User.customUrl"]);
+	// .$assertType<FindPlacement>();
+}
+
+export async function findPlacementsOfMonth(
+	args: Pick<Tables["XRankPlacement"], "mode" | "region" | "month" | "year">,
+) {
+	return await selector()
+		.where("XRankPlacement.mode", "==", args.mode)
```

Comment:
```suggestion
		.where("XRankPlacement.mode", "=", args.mode)
```

Maybe the == works too? But for consistency lets use "=" everywhere